.github/workflows/build.yml

```diff
@@ -91,10 +91,24 @@ jobs:
       run: |
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
         docker push antrea/antrea-ubuntu-simulator:latest
+    
+
+  check-windows-changes:
+     runs-on: [ubuntu-latest]
+     outputs:
+       all: ${{ steps.changed_window_files.outputs.all}}
+     steps:
+     - uses: actions/checkout@v2
+       with:
+        fetch-depth: 0
+     - name: Get changed files
+       id: changed_window_files
+       run: |
+         echo "::set-output name=all::$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} | grep -E -l "//go:build windows" *.go | xargs)"
```

Comment:
I think you need to exclude *`_windows_test.go` file since go build will not compile them. please chech `go help build`